### DISCUSSION TITLE
Suggestion for single sample and merging objects vignettes

### DISCUSSION THREAD

**Discussion by wong-nw** [stuart-lab/signac#1629] [Ideas]
Overall, Signac covers everything that I'm trying to do, and I appreciate how straightforward many of the tools are. But I'm working on a project with multiple samples that need to be merged together, and I started with the [PBMC vignette](https://stuartlab.org/signac/articles/pbmc_vignette) before moving on to the [merge samples vignette](https://stuartlab.org/signac/articles/merging). I didn't realize that many of the steps I took in the PBMC vignette for the individual samples would need to be repeated for the multi-sample runs, due to the use of the common peak set. Would it be possible to put a warning/disclaimer on the PBMC vignette for other users who might run into the same issue with order of operations?

Thank you!

-Nathan

**Comment by timoast** [comment]
Hi Nathan, appreciate your feedback, we can try to make things more clear.
